lib/cassia/response_handlers/get_connected_devices_router.rb

```diff
@@ -0,0 +1,29 @@
+module Cassia
+  module ResponseHandlers
+    class GetConnectedDevicesRouter
+      def initialize(access_controller, router: )
+        @access_controller = access_controller
+        @router = router
+      end
+
+      def handle(response)
+        if response.success?
+          handle_success(response)
+        else
+          handle_failure(response)
+        end
+        response.success?
+      end
+
+      private
+
+      def handle_success(response)
+        @router.connected_devices = response.body["nodes"]
```

Comment:
I think you need to iterate over `response.body["nodes"]` and create a new device for each item in `"nodes"`